environment.py

```diff
@@ -137,13 +142,23 @@ def remote_cmd(cmd, host=None, ignore_rc=False, **kwargs):
         # the 'context' object can basically hold whatever we want.
         # if we stash the result from Ansible, we can inspect it or log it
         # later
-        context.result = ansible.runner.Runner(
-                 module_name=cmd,
-                 inventory=inventory,
-                 pattern=host,
-                 #remote_user=remote_user,
-                 **kwargs
-        ).run()
+        if async:
+            context.result, poller = ansible.runner.Runner(
+                     module_name=cmd,
+                     inventory=inventory,
+                     pattern=host,
+                     #remote_user=remote_user,
+                     **kwargs
+            ).run_async(context.test_timeout)
+            return poller
+        else:
+            context.result = ansible.runner.Runner(
+                     module_name=cmd,
+                     inventory=inventory,
+                     pattern=host,
+                     #remote_user=remote_user,
+                     **kwargs
+            ).run()
```

Comment:
nit:  The indents in these lines look off.  Try using `flake8` or `pep8` to detect style issues.